src/api/config/speedsettings.rs

```diff
@@ -246,8 +246,14 @@ impl SpeedSettings {
     false
   }
 
-  const fn fast_scene_detection_preset(speed: usize) -> bool {
-    speed == 10
+  const fn fast_scene_detection_preset(speed: usize) -> SceneDetectionSpeed {
+    if speed <= 6 {
```

Comment:
Can we expose this option to CLI also, like users could explicitly set different speed detection mode irrespective of default modes? I think that might be a good addition